pipeline/utils.py

```diff
@@ -0,0 +1,182 @@
+import math
+import os
+import random
+from typing import List
+
+import numpy as np
+import torch
+from torch.optim.lr_scheduler import LambdaLR
+
+import utils
+from utils import print_rank_0
+
+def get_cache_dir():
+    DEFAULT_HF_HOME = "~/.cache/huggingface"
+    cache_dir = os.environ.get("HF_HOME", DEFAULT_HF_HOME)
+
+    return cache_dir
+
```

Comment:
```suggestion


def get_cache_dir():
    DEFAULT_HF_HOME = "~/.cache/huggingface"
    cache_dir = os.environ.get("HF_HOME", DEFAULT_HF_HOME)

    return cache_dir


```

Following https://peps.python.org/pep-0008/#blank-lines